### PR TITLE
PEP 637: Integrate feedback from GvR

### PR DESCRIPTION
**Pull request by stefanoborini**
Integrates feedback from GvR. 

Note: this PR is built on top of #1619

### CODE REVIEW

**Comment by gvanrossum** [comment]
Please resolve the conflict that appeared now #1619 is merged.

**Comment by stefanoborini** [comment]
@gvanrossum should be solved. Strange that I ended up with conflicts, as I worked on top of the other PR. No big deal.

File: pep-0637.rst [hunk 1]
```diff
@@ -536,6 +581,7 @@ need to be implemented to support the extended call. We propose
 - ``PyObject_SetItemWithKeywords(PyObject *o, PyObject *key, PyObject *value, PyObject *kwargs)``
 - ``PyObject_GetItemWithKeywords(PyObject *o, PyObject *key, PyObject *kwargs)``
 
+<<<<<<< HEAD:pep-0637.rst
```
Comment (JelleZijlstra) [inline pep-0637.rst]: Looks like a poorly resolved merge conflict

Comment (stefanoborini) [reply pep-0637.rst]: @JelleZijlstra it has been fixed already. Not sure why one travis is hanging on queued though.

Comment (JelleZijlstra) [reply pep-0637.rst]: Sorry for the noise.

If you click through to https://travis-ci.com/github/python/peps/jobs/391286293 it actually passed. I've seen pretty often that Travis status updates don't go through to GitHub for some reason.

Comment (stefanoborini) [reply pep-0637.rst]: I try to kick it again. One sec.

**Review by gvanrossum — APPROVED** [review]

**Pull request merged**